scientific/spectroscopy/specpr.ksy

```diff
@@ -0,0 +1,254 @@
+meta:
+  id: specpr
+  file-extension: spec
+  endian: be
+  license: unlicense

```

Comment:
This is not a valid [SPDX expression](https://spdx.org/licenses/). `Unlicense` is.